.github/workflows/python-app.yml

```diff
@@ -0,0 +1,42 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python application
+
+on:
```

Comment:
should we follow [this pattern](https://github.com/chanzuckerberg/single-cell-explorer/blob/main/.github/workflows/push_tests.yml#L3-L7)?

Reply:
Good idea! Thanks